java/src/test/resources/source/do_not_format_me/UnnecessaryModifier/NestingRecord_Issue846_Post.java

```diff
@@ -1,25 +1,136 @@
-public record SomeRecord(int i) {
-	public interface SomeInterface {
-	}
+public class Container {
+	public abstract class SomeClass {
```

Comment:
Isn't this duplicating the cases covered in `TestUnnecessaryModifierCases`?

I also feel uncomfortable accumulating these many cases in a single `pre-post` case. (e.g. given file is named `NestingRecord`, as it focus on given specific case).